content/articles/dnssec.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: DNSSEC
+excerpt: DNSimple provides full support for DNSSEC in our authoritative name servers, including signing of zones that are registered outside of DNSimple.
+categories:
+- DNS
+---
+
+# DNSSEC
+
```

Comment:
We should add the "private beta" warning. Or public one?

https://github.com/dnsimple/dnsimple-support/blame/2186aff62ebacf848d1ba83b12dbeeecbfd52283/content/articles/github-sync.markdown#L10-L12
https://github.com/dnsimple/dnsimple-support/blame/d40716fdd6c24651f917475f437f15bdb8557c8b/content/articles/ordering-lets-encrypt-certificate.markdown#L10-L12